src/main/java/uk/gov/hmcts/probate/service/notification/GrantOfRepresentationPersonalisationService.java

```diff
@@ -127,8 +129,10 @@ private String parseDelimiters(String value) {
 
     private String getWillReferenceNumber(CaseData data) {
         for (CollectionMember<ScannedDocument> document : data.getScannedDocuments()) {
-            if (document.getValue().getSubtype() != null && document.getValue().getSubtype()
-                .equalsIgnoreCase(DOC_SUBTYPE_WILL)) {
+            if (document.getValue().getSubtype() != null
```

Comment:
do you still want to check the subtype is not null here even if looking only at type will in the or?

Reply:
good point, left it in as it was there before but thinking about it, in case 1 its checked subtype is will already and case 2 there shouldn't be any subtype nulls with type will, will see if can remove it